scrapper.py

```diff
@@ -1,6 +1,14 @@
 """
 Scrapper implementation
 """
+import json
```

Comment:
all imports should be grouped into:

1. standard Python imports
2. 3rd party imports
3. project imports

all imports should be in alphabetic order